client/firewall/nftables/manager_linux.go

```diff
@@ -326,7 +343,7 @@ func (m *Manager) getOrCreateSet(
 	rawIP []byte,
 	name string,
 ) (*nftables.Set, bool, error) {
-	ipset, err := m.conn.GetSetByName(table, name)
+	ipset, err := m.rConn.GetSetByName(table, name)
```

Comment:
using rCoon here

Reply:
rConn because sConn used only to add/remove IP addresses to set